src/lib/components/input-and-result.svelte

```diff
@@ -4,7 +4,7 @@
 
   export let events: HistoryEventWithId[];
 
-  $: inputAndResult = getWorkflowStartedAndCompletedEvents(events);
+  let inputAndResult = getWorkflowStartedAndCompletedEvents(events);
```

Comment:
Was the reactivity statement here messing with the page or was this for another reason? Mostly curious for my own knowledge